### PR TITLE
[linters] task: tabify missed files, rename yml to yaml

### PR DESCRIPTION
**Pull request by Jaguar0625**

### CODE REVIEW

File: sdk/python/symbolchain/symbol/ExtendedTypeParsingRules.py [hunk 1]
```diff
@@ -6,8 +6,7 @@
 def extend_type_parsing_rules(symbol_type_converter, base_type_parsing_rules=None):
 	builder = TypeParserBuilder('sc', symbol_type_converter, base_type_parsing_rules)
 
-	# note, some of those below could be automatically generated,
-	# but right now symbol.yml contains also state-related objects
+	# note, some of those below could be automatically generated
```
Comment (Jaguar0625) [inline sdk/python/symbolchain/symbol/ExtendedTypeParsingRules.py:9]: @gimre-xymcity - not sure what this comment should say as there is no symbol.y(a)ml

File: linters/scripts/find_space_indents.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -ex
+
+! rg --files-with-matches '^  ' "$(git rev-parse --show-toplevel)" | grep -vE ".json|.md|.LESSER|.eslintrc|.yaml|.txt"
```
Comment (Jaguar0625) [inline linters/scripts/find_space_indents.sh:5]: @Wayonb - not sure if we should integrate this into CI as we currently don't have place to run something across all files in repo and/or that ALWAYS runs 🤔

File: sdk/javascript/src/nem/external/tweetnacl-nacl-fast-keccak.js [hunk 3]
```diff
@@ -12,9 +12,9 @@ const { keccak_512 } = require('js-sha3');
 // See for details: http://tweetnacl.cr.yp.to/
```
Comment (gimre-xymcity) [inline sdk/javascript/src/nem/external/tweetnacl-nacl-fast-keccak.js]: as mentioned, I would not touch this file at all...

File: sdk/python/symbolchain/symbol/ExtendedTypeParsingRules.py (hunk 1 above)
Comment (gimre-xymcity) [reply sdk/python/symbolchain/symbol/ExtendedTypeParsingRules.py:9]: it's fine, we could generate (some/ most) of the rules based on AST iterate through all things, 
 - for pods, flags and enums we could probably always fire rule
 - for structs - only those that are in hints of transactions

File: sdk/javascript/src/nem/external/tweetnacl-nacl-fast-keccak.js (hunk 3 above)
Comment (Jaguar0625) [reply sdk/javascript/src/nem/external/tweetnacl-nacl-fast-keccak.js]: i think it's low risk (most diffs can ignore whitespace) and we did same to external one in py 😅

File: linters/scripts/find_space_indents.sh (hunk 2 above)
Comment (Wayonb) [reply linters/scripts/find_space_indents.sh:5]: Make sense.  Is the rg tool part of the new docker images?  If so we could add another step or as part of lint.

File: sdk/javascript/src/nem/external/tweetnacl-nacl-fast-keccak.js (hunk 3 above)
Comment (gimre-xymcity) [reply sdk/javascript/src/nem/external/tweetnacl-nacl-fast-keccak.js]: 😬

**Review by gimre-xymcity — APPROVED** [review]

**Pull request merged**